### PR TITLE
Fix some UG details and update README

### PR DESCRIPTION
**Pull request by sun-ruiheng**
- added some newer command details to the UG
- made the main README a much shorter version of our updated UG

### CODE REVIEW

**Review by gavingoh99 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -406,15 +399,20 @@ _Details coming soon ..._
...
+| **Find**             | `find KEYWORD/ [KEYWORD]`<br> e.g., `find n/ James n/ T t/ friend t/ rich`                                                                                                                      |
+| **Add Order**        | `addorder INDEX d/DATE r/REMARK`<br> e.g., `addorder 1 d/ 2020-01-01 r/ 100 chicken wings`                                                                                                      |
+| **List Orders**      | `listorder INDEX`                                                                                                                                                                               |
+| **Delete Order**     | `deleteorder INDEX o/ORDER_INDEX`<br> e.g., `deleteorder 2 o/1`                                                                                                                                 |
+| **Add Favourite**    | `addfav [i/INDICES]`<br> e.g., `addfav i/1,2`                                                                                                                                                   |
```
Comment (guanquann) [inline docs/UserGuide.md:412]: I think there should be no `[]`, `[]` is used for optional parameters

File: docs/UserGuide.md [hunk 2]
```diff
@@ -406,15 +399,20 @@ _Details coming soon ..._
...
+| **Show Favourites**  | `showfav`                                                                                                                                                                                       |
+| **Remove Favourite** | `removefav [i/INDICES]`<br> e.g., `removefav i/2`                                                                                                                                               |
```
Comment (guanquann) [inline docs/UserGuide.md:414]: Similar issue as above

**Comment by guanquann** [comment]
LGTM

**Pull request merged**